roles/app/templates/application.yml.j2

```diff
@@ -14,3 +14,11 @@ LOCALE: {{ locale }}
 CURRENCY: {{ currency }}
 # Spree zone.
 CHECKOUT_ZONE: {{ checkout_zone }}
+
+GOOGLE_MAPS_API_KEY: {{ google_maps_api_key }}
+
+S3_ACCESS_KEY: {{ s3_access_key }}
```

Comment:
I'm not super familiar with the AWS access management, but it could be that developers have access to the images (public data), but not to the backups (highly sensitive data). And I don't know how people are organising there AWS keys and buckets. If I don't make sense, leave it. You could add a comments to the application.yml.